webviz_subsurface_components/py_expression_eval.py

```diff
@@ -0,0 +1,483 @@
+"""
+Copyright (c) 2021- Equinor ASA
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+
+This is a modification of py-expression-eval created by AxiaCore. 
+
+Moifications are done to obtain simplifed functionality for itended usage.
+
+Thanks to AxiaCore!
+
+`py-expression-eval:`
+
+Author: AxiaCore S.A.S. http://axiacore.com
+GitHub: https://github.com/AxiaCore/py-expression-eval/
+
+Based on js-expression-eval, by Matthew Crumley (email@matthewcrumley.com, http://silentmatt.com/)
+https://github.com/silentmatt/js-expression-eval
+
+Ported to Python and modified by Vera Mazhuga (ctrl-alt-delete@live.com, http://vero4ka.info/)
+
+You are free to use and modify this code in anyway you find useful. Please leave this comment
+in the code to acknowledge its original source. If you feel like it, I enjoy hearing about
+projects that use my code, but don't feel like you have to let me know or ask permission.
+"""
+# pylint: skip-file
+
+from __future__ import division
+
+import re
+
+import numpy as np
+
+TNUMBER = 0
+TOP1 = 1
+TOP2 = 2
+TVAR = 3
+TFUNCALL = 4
+
+
+class Token:
+    def __init__(self, type_, index_, prio_, number_):
+        self.type_ = type_
+        self.index_ = index_ or 0
+        self.prio_ = prio_ or 0
+        self.number_ = number_ if number_ != None else 0
+
+    def toString(self):
```

Comment:
```suggestion
    def to_string(self):
```
Stick with camel case only on Python classes?